integrations/FacebookPixel/browser.js

```diff
@@ -148,10 +148,10 @@ class FacebookPixel {
     if (properties) {
       products = properties.products;
       quantity = properties.quantity;
-      category = properties.category;
+      category = properties.category || "";
       prodId = properties.product_id || properties.id || properties.sku || "";
-      prodName = properties.product_name;
-      value = properties.value;
+      prodName = properties.product_name || properties.name || "";
+      value = properties.value || properties.price;
```

Comment:
We are mapping price to both `value` and `price` in case `value` is not present. Is that the correct behaviour

Reply:
In cloud mode, there is no price mapped to price but value or price is mapped to value.

Reply:
We have to refactor this when destination mapping versioning mgmt is  in place

Reply:
Resolving this conversation on @utsabc's behalf as the PR is now merged.